Simulation/Headers/Components/com_Physics.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <glm/glm.hpp>
+
+#include "ECS/Types.h"
+
+#include "Algorithms/Physics.h"
+
+#include "Components/com_Transform.h"
+
+namespace Component
+{
+	class com_Physics
+	{
+	public:
+		com_Physics() {};
+		com_Physics( 
+			float mass, float invmass, float res,
+			glm::vec3 com, glm::mat4 inertia,
+			glm::vec3 vel = glm::vec3(0, 0, 0),
+			glm::vec3 acc = glm::vec3(0, 0, 0),
+			glm::vec3 rotvel = glm::vec3(0, 0, 0),
+			glm::vec3 rotacc = glm::vec3(0, 0, 0));
```

Comment:
Can you make so that these are not default set, and are instead list set
So take them out of the constructor and ) : vel(glm::vec3(0) etc

Reply:
Nevermind, just seen you have done that. However, can you take the vel, acc, rotvel and rotacc out of the constructor

Reply:
absolutely not, I refuse


yes